.github/workflows/tests_and_cov.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           python-version: 3.11
 
       - uses: syphar/restore-virtualenv@v1
-         id: cache-virtualenv
+        id: cache-virtualenv
 
       - name: Run pytest
         run: pytest --cov --test-group-count 6 --test-group=${{ matrix.group }} --test-group-random-seed=12345 --rootdir=/home/runner/work/scout
```

Comment:
Have you tried with `-n auto`?

One could also consider a beefier runner with more CPU, but it will cost a bit more.

Reply:
How do I use -n auto, do you have any example?

Reply:
https://github.com/Clinical-Genomics/cg/commit/a480578f2b6d9f2d8a0d1a88c4398f079a898ed8

Reply:
Mmm I don't know about that beefy linux. I like the idea, but feels like it takes a long time before the test start, since you are waiting for a beefy worker to pick the job

Reply:
Both the -n auto and -n logical option make tests fail. I'm not sure how they work:

```
ERROR tests/server/blueprints/variants/test_variants_views.py::test_mei_variants - scout.exceptions.database.IntegrityError: Institute test_institute already exists in database
ERROR tests/server/blueprints/variants/test_variants_views.py::test_bulk_reset_dismiss_variants - pymongo.errors.OperationFailure: index not found with name [category_verb_subject], full error: {'ok': 0.0, 'errmsg': 'index not found with name [category_verb_subject]', 'code': 27, 'codeName': 'IndexNotFound'}
ERROR tests/commands/update/test_update_individual_cmd.py::test_update_individuals_key_value[splice_junctions_bed] - scout.exceptions.database.IntegrityError: Institute cust000 could not be found
```